database/migrations/2022_02_09_143651_create_taskcrudapps_table.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class CreateTaskcrudappsTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('taskcrudapps', function (Blueprint $table) {
```

Comment:
always table name will be `snake_case` with plural name. If your model name is `TaskCrudApp`, table name will be `task_crud_apps`